third-party/sysdeps/common/sqlite3/CMakeLists.txt

```diff
@@ -70,6 +70,7 @@ if(NOT WIN32)
     install(FILES ${CMAKE_CURRENT_BINARY_DIR}/libsqlite3.so TYPE LIB)
 endif()
 install(FILES "${SOURCE_DIR}/sqlite3.h" TYPE INCLUDE)
+install(FILES "${SOURCE_DIR}/sqlite3ext.h" TYPE INCLUDE)
```

Comment:
Thanks, this fixes https://github.com/ROCm/TheRock/issues/162. Then with a few venv patches for https://github.com/ROCm/TheRock/issues/158 and https://github.com/ROCm/TheRock/issues/161, I can build TheRock's `all` target on my Ubuntu 22.04 system (outside of Docker).

Reply:
Nice. Sorry I missed this originally.